src/api/generate-xlsx/map-application-to-XLSX/map-declarations/map-modern-slavery-fields/index.test.ts

```diff
@@ -0,0 +1,38 @@
+import mapModernSlaveryFields from '.';
+import DECLARATIONS_FIELD_IDS from '../../../../constants/field-ids/insurance/declarations';
+import { DECLARATIONS_FIELDS } from '../../../../content-strings/fields/insurance/declarations';
+import xlsxRow from '../../helpers/xlsx-row';
+import mapYesNoField from '../../helpers/map-yes-no-field';
+import mapCannotAdhereConditionalReason from './map-cannot-adhere-conditional-reason';
+import mapOffensesConditionalReason from './map-offenses-conditional-reason';
+import mapAwareOfConditionalReason from './map-aware-of-conditional-reason';
+import { mockApplication } from '../../../../test-mocks';
+
+const {
+  MODERN_SLAVERY: { WILL_ADHERE_TO_ALL_REQUIREMENTS, HAS_NO_OFFENSES_OR_INVESTIGATIONS, IS_NOT_AWARE_OF_EXISTING_SLAVERY },
+} = DECLARATIONS_FIELD_IDS;
+
+const CONTENT_STRINGS = DECLARATIONS_FIELDS.MODERN_SLAVERY;
+
+const {
+  declaration: { modernSlavery },
+} = mockApplication;
+
+describe('api/generate-xlsx/map-application-to-xlsx/map-declarations/map-modern-slavery-fields', () => {
+  it('should return an array of mapped declaration fields', () => {
```

Comment:
Can we please add an assertion for invalid argument or exception being thrown when calling `mapModernSlaveryFields`. Thanks

Reply:
Missed this one.

Reply:
Forgot to reply - same reply as here https://github.com/UK-Export-Finance/exip/pull/3446/files#r1907205946